cit/templates/index.html~

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
html~ -- any file with next *.*~ pattern should be avoided and pushed to .gitignoere